source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -243,17 +243,16 @@ IClientAndServiceBuilder IClientAndServiceBuilder.WithProxy()
 
         public LatestClientAndLatestServiceBuilder WithProxy()
         {
-            this.proxyFactory = () =>
-            {
-                var options = new HttpProxyOptions();
-                var loggerFactory = new SerilogLoggerFactory(new SerilogLoggerBuilder().Build());
-
-                return new HttpProxyService(options, loggerFactory);
-            };
-
+            this.proxyFactory = new ProxyFactory();
             return this;
         }
 
+        private bool ProxyDelaySendingSectionsOfHttpHeaders()
```

Comment:
This is sort of hidden away, I wonder if the builder should expose some concept of enable setting on the proxy that stress the halibut client.